app/decorators/phone_configuration_decorator.rb

```diff
@@ -0,0 +1,6 @@
+PhoneConfigurationDecorator = Struct.new(:phone_configuration) do
+  def default_msg
```

Comment:
What about `default_number_message` here? `default_msg` makes me thing this is the message you display by default somewhere.